components/FishForm.js

```diff
@@ -130,13 +125,41 @@ class FishForm extends Component {
 }
 
 const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    paddingTop: 50,
+    marginTop: "10%"
+  },
   label: {
```

Comment:
When we finalized our styling, we can move these methods to another file and component will reference styling in this file.  This can be implemented in future updates.

Reply:
It looks like this is how we would do it: https://www.reactnative.guide/8-styling/8.3-separating-styles-from-component.html

But I agree, let's get the styling finalized first, then we can refactor.